src/main/resources/local-dev/toolbox-local.xh.io.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
We can see if this gets flagged by any automated secret sniffing routines.  

Wondering if we should put a word like "test" or "self-signed" in the filenames to help drive home the point that these are not real credentials.  I know it says "local" maybe that's enough.  (To be clear, don't think we need the test domain name to be any different, just talking about these three filenames)

Reply:
added 'self-signed' to file names.